Formula/numpy.rb

```diff
@@ -38,9 +46,15 @@ def install
 
     xy = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
     ENV.prepend_create_path "PYTHONPATH", Formula["cython"].opt_libexec/"lib/python#{xy}/site-packages"
+    ENV.prepend_path "PYTHONPATH", libexec/"vendor/lib/python#{xy}/site-packages"
```

Comment:
```suggestion
    ENV.prepend_path "PYTHONPATH", libexec/"vendor"/Language::Python.site_packages("python3")
```
Is this not only needed at build-time?

Reply:
It should only be necessary at build-time, but being a complete Python n00b, I cargo-culted this off the `yle-dl` formula. Is there a better solution?

Reply:
OK, using `buildpath/"temp"` instead of `libexec/"vendor"` seems to work.